src/legacy/ui/public/vis/editors/default/_agg_params.scss

```
@@ -10,3 +10,12 @@
   color: $textColor;
   background-color: $backgroundColor;
 }
+
+.visEditorAggParam--inline {
+  display: inline-block;
+  width: 50%;
+}
+
+.visEditorAggParam--inline-size {
+  padding-left: $vis-editor-agg-editor-spacing;
```

We can start phasing out these custom variables and just use EUI variables:

```suggestion
  margin-left: $euiSizeS;
```

Also change to margin so it doesn't affect the size of the control.

---

good point, done! 👍